RTB_OP_CODES.s

```diff
@@ -113,13 +113,13 @@ TPMODE		CAF	ONE		# MODE IS TP.
 		TS	MPAC		# AND SKIP ON OVERFLOW.
 		TC	Q
 
-		INDEX	A		# OVERFLOW UNCORRECT AND IN MSU.
+		INDEX	A		# OVERFLOW INCORRECT AND IN MSU.
```

Comment:
`UNCORRECT` is used in [source](http://www.ibiblio.org/apollo/ScansForConversion/Luminary099/1398.jpg)